packages/react-widgets/src/NumberPicker.tsx

```diff
@@ -16,29 +16,29 @@ import { notify } from './util/widgetHelpers'
 
 // my tests in ie11/chrome/FF indicate that keyDown repeats
 // at about 35ms+/- 5ms after an initial 500ms delay. callback fires on the leading edge
-function createInterval(callback) {
-  let fn
-  var id,
-    cancel = () => clearTimeout(id)
+function createInterval(callback : ()=> void) {
+  let fn : ()=> void;
+  let id : number;
+  const cancel = () => clearTimeout(id);
 
-  id = setTimeout(
+  id = window.setTimeout(
     (fn = () => {
-      id = setTimeout(fn, 35)
+      id = window.setTimeout(fn, 35)
       callback() //fire after everything in case the user cancels on the first call
     }),
     500,
-  )
+  );
 
   return cancel
 }
 
-function clamp(value, min, max) {
+function clamp(value : number | string | null | undefined, min : number , max: number) {
```

Comment:
the string shouldn't be in here, what is requiring that?

Reply:
the clamp function itself was testing for '', changed to `(value as any) === ''`